index.html

```diff
@@ -5,26 +5,26 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="style.css" />
-    <title>Document</title>
+    <title>Fabien Portfolio</title>
   </head>
   <body>
     <div class="container">
-      <header class="top">
+      <section class="top">
         <a class="logo" href="#">Fabien</a>
         <nav class="ham">
           <img class="hami" src="./img/Union.png" alt="hamburger img" />
         </nav>
         <nav class="nav-bar">
-          <h4 class="adj">Portfolio</h4>
-          <h4 class="adj">About</h4>
-          <h4 class="adj">Contact</h4>
-          <img class="letter" src="./img/Union1.png" alt="mail image" />
+          <a class="adj" href="">Portfolio</a>
+          <a class="adj" href="">About</a>
+          <a class="adj" href="">Contact</a>
+          <a href=""><img class="letter" src="./img/Union1.png" alt="mail image" /></a>
         </nav>
-      </header>
+      </section>
```

Comment:
- [ ] Good job working on the changes requested by the previous reviewer. However, I noticed you changed the toolbar section wrapper from a `<header>` tag to a `<section>` tag. For semantic purposes, kindly make the wrapper a `<header>` tag. You can read more about the use of a `<header>` tag [here](https://www.w3schools.com/tags/tag_header.asp)

Reply:
so can I have 2 header tags then? @phelian23

Reply:
ok so I can have more than 1 header, once it is not in a footer, address or in another header, so there should be no problems now